### PR TITLE
feat(sendy): Count Subscribers / Create Campaign / Delete Subscriber  / Get Brand Lists / Get Brands / Get Subscription Status /Subscribe /  Unsubscribe Action  

### PR DESCRIPTION
**Pull request by joeworkman**
This PR adds a new piece for Sendy. https://sendy.co

I have created end points for every documented API. https://sendy.co/api

I have tested each action pretty thoroughly and verified on my production instance of Sendy. 

I look forward to any feedback that you may have. 

ANNOUNCEMENT=true

### CODE REVIEW

File: packages/pieces/sendy/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { deleteAction } from "./lib/actions/delete";
+import { statusAction } from "./lib/actions/status";
+import { countAction } from "./lib/actions/count";
+import { createCampaignAction } from "./lib/actions/createCampaign";
+
+export const sendy = createPiece({
+	displayName             : "Sendy",
+	auth                    : sendyAuth,
+	minimumSupportedRelease : '0.9.0',
+	logoUrl                 : "https://zapier-images.imgix.net/storage/developer/242e928e4b24ea7d0823b105c3e2bcb0_3.png",
```
Comment (abuaboud) [inline packages/pieces/sendy/src/index.ts]: Please change it to

https://cdn.activepieces.com/pieces/sendy.png

File: packages/pieces/sendy/src/lib/actions/getLists.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import { getLists } from "../api";
+import { sendyAuth } from "../auth";
+
+export const getListsAction = createAction({
+	name        : 'getLists',
+	auth        : sendyAuth,
+	displayName : 'Get Lists',
+	description : 'Get the Lists for a Brand',
+	props       : {
+		// brandId: Property.Dropdown({
```
Comment (abuaboud) [inline packages/pieces/sendy/src/lib/actions/getLists.ts]: Is this commented by mistake? If not can you please remove it

File: packages/pieces/sendy/project.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    },
+    "lint": {
+      "executor": "@nx/eslint:lint",
+      "outputs": [
+        "{options.outputFile}"
+      ],
+      "options": {
+        "lintFilePatterns": [
+          "packages/pieces/sendy/**/*.ts",
+          "packages/pieces/sendy/package.json"
```
Comment (abuaboud) [inline packages/pieces/sendy/project.json]: Can you please remove that to fix the lint issue.

I tried to fix it but didn't have permission to edit this PR

**Comment by abuaboud** [comment]
@allcontributors please add @joeworkman for plugin

File: packages/pieces/sendy/src/lib/actions/status.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import { createAction, Property, Validators } from "@activepieces/pieces-framework";
+import { status } from "../api";
+import { buildListDropdown } from "../props";
+import { sendyAuth, SendyAuthType } from "../auth";
+
+export const statusAction = createAction({
+	name        : 'status',
+	auth        : sendyAuth,
+	displayName : 'Subscription Status',
```
Comment (abuaboud) [inline packages/pieces/sendy/src/lib/actions/status.ts:9]: Can you please rename it to "Get Subscription Status"

File: packages/pieces/sendy/src/lib/actions/subscribe.ts [hunk 5]
```diff
@@ -0,0 +1,73 @@
+import { createAction, Property, Validators } from "@activepieces/pieces-framework";
+import { subscribe } from "../api";
+import { buildListDropdown } from "../props";
+import { sendyAuth, SendyAuthType } from "../auth";
+
+export const subscribeAction = createAction({
+	name        : 'subscribe',
+	auth        : sendyAuth,
+	displayName : 'Subscribe',
```
Comment (abuaboud) [inline packages/pieces/sendy/src/lib/actions/subscribe.ts:9]: Can you please rename to `Add Subscriber`

File: packages/pieces/sendy/src/lib/actions/unsubscribe.ts [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import { createAction, Property, Validators } from "@activepieces/pieces-framework";
+import { unsubscribe } from "../api";
+import { buildListDropdown } from "../props";
+import { sendyAuth, SendyAuthType } from "../auth";
+
+export const unsubscribeAction = createAction({
+	name        : 'unsubscribe',
+	auth        : sendyAuth,
+	displayName : 'Unsubscribe',
```
Comment (abuaboud) [inline packages/pieces/sendy/src/lib/actions/unsubscribe.ts:9]: Can you please rename it to `Remove Subscriber`

File: packages/pieces/sendy/src/lib/actions/getLists.ts [hunk 7]
```diff
@@ -0,0 +1,32 @@
+import { createAction, Property } from "@activepieces/pieces-framework";
+import { getLists } from "../api";
+import { sendyAuth } from "../auth";
+
+export const getListsAction = createAction({
+	name        : 'getLists',
+	auth        : sendyAuth,
+	displayName : 'Get Lists',
```
Comment (abuaboud) [inline packages/pieces/sendy/src/lib/actions/getLists.ts]: `Get Brand Lists`

**Review by abuaboud — CHANGES_REQUESTED** [review]

File: packages/pieces/sendy/src/lib/actions/count.ts [hunk 8]
```diff
@@ -0,0 +1,27 @@
+import { createAction, Property } from "@activepieces/pieces-framework";
+import { count } from "../api";
+import { buildListDropdown } from "../props";
+import { sendyAuth, SendyAuthType } from "../auth";
+
+export const countAction = createAction({
+	name        : 'count',
+	auth        : sendyAuth,
+	displayName : 'Active Subscriber Count',
```
Comment (abuaboud) [inline packages/pieces/sendy/src/lib/actions/count.ts:9]: Count Active Subscriber

File: packages/pieces/sendy/src/lib/actions/unsubscribe.ts (hunk 6 above)
Comment (joeworkman) [reply packages/pieces/sendy/src/lib/actions/unsubscribe.ts:9]: I don't agree here as well. This does not remove a subscriber. That would be delete. I think that this needs to be named unsubscribe. There is a difference.

File: packages/pieces/sendy/src/lib/actions/subscribe.ts (hunk 5 above)
Comment (joeworkman) [reply packages/pieces/sendy/src/lib/actions/subscribe.ts:9]: I don't agree with this. The subscriber may already exist but is just subscribing to a new list.

File: packages/pieces/sendy/project.json (hunk 3 above)
Comment (joeworkman) [reply packages/pieces/sendy/project.json]: Sure. Interesting that I did not add this. Maybe my tooling did?

**Comment by joeworkman** [comment]
I made all of the required updates, minus the subscribe/unsubscribe rename. I noted my reasons for disagreeing with those above.

**Comment by abuaboud** [comment]
I don't have permission to push to this branch (I opened new PR with minor lint issue) so this should be marked as merged.
https://github.com/activepieces/activepieces/pull/3165


Thank you a lot!

**Pull request merged**